identity/src/keypair.rs

```diff
@@ -822,25 +830,24 @@ mod tests {
     }
 
     #[test]
-    #[cfg(all(
-        feature = "ecdsa",
-        feature = "secp256k1",
-        feature = "ed25519",
-        feature = "rsa",
-        feature = "peerid"
-    ))]
+    #[cfg(feature = "peerid")]
     fn keypair_from_protobuf_encoding() {
-        // E.g. retrieved from an IPFS config file.
-        let base_64_encoded = "CAESQL6vdKQuznQosTrW7FWI9At+XX7EBf0BnZLhb6w+N+XSQSdfInl6c7U4NuxXJlhKcRBlBw9d0tj2dfBIVf6mcPA=";
-        let expected_peer_id =
-            PeerId::from_str("12D3KooWEChVMMMzV8acJ53mJHrw1pQ27UAGkCxWXLJutbeUMvVu").unwrap();
```

Comment:
This pull request removes the `keypair_from_protobuf_encoding` `ed25519` test. In my eyes this is sufficiently covered with the more elaborate round-trip test introduced with https://github.com/libp2p/rust-libp2p/pull/4228.